extensions/authclient/BaseOAuth.php

```diff
@@ -397,7 +397,7 @@ protected function composeUrl($url, array $params = [])
     /**
      * Saves token as persistent state.
      * @param OAuthToken $token auth token
-     * @return static self reference.
+     * @return $this
```

Comment:
the comment should stay: `$this self reference.`
Also `self reference` could be improved to say a bit more.